UhooiPicBook/Modules/MonsterList/Views/MonsterCollectionViewCell.swift

```diff
@@ -26,6 +26,13 @@ final class MonsterCollectionViewCell: UICollectionViewCell {
             newValue.text = nil
         }
     }
+    
+    // MARK: View Life-Cycle Methods
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        iconImageView.image = nil
```

Comment:
個人的な好みだけど、プロパティのアクセスは常に `self.` を付けてほしいです 🙏 

```suggestion
        self.iconImageView.image = nil
```